annotation-processing/src/main/java/processor/FastProcessor.java

```diff
@@ -0,0 +1,63 @@
+package processor;
+
+import com.google.auto.service.AutoService;
+
+import javax.annotation.processing.*;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.tools.JavaFileObject;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+@SupportedAnnotationTypes("processor.FastType")
+@SupportedSourceVersion(SourceVersion.RELEASE_8)
+@AutoService(Processor.class)
+public class FastProcessor extends AbstractProcessor {
+
+    @Override
+    public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+        for (TypeElement annotation : annotations) {
+            Set<? extends Element> annotatedElements = roundEnv.getElementsAnnotatedWith(annotation);
+            Map<String, String> nameClassMap = new HashMap<>();
+            annotatedElements.forEach(clazz->{
+                String annotationValue = clazz.getAnnotation(FastType.class).name();
+                if (annotationValue.equals("")){
+                    annotationValue = clazz.getSimpleName().toString();
+                }
+                if (nameClassMap.containsKey(annotationValue)) throw new RuntimeException("Duplicate name value for @FastType annotation");
+                nameClassMap.put(annotationValue, clazz.toString());
+            });
+            try {
+                buildCreatorClass(nameClassMap);
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
Never deal with exceptions such way. In case of annotations processor you need to use processingEnv.getMessager() to report errors